enzyme/test/Enzyme/ForwardMode/vector_reduce_fadd.ll

```diff
@@ -0,0 +1,45 @@
+; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -mem2reg -instsimplify -simplifycfg -S | FileCheck %s
+
+define float @tester1(<4 x float> %input) {
+entry:
+  %unord = call reassoc float @llvm.vector.reduce.fadd.v4f32(float -0.0, <4 x float> %input)
```

Comment:
Can you make the first arg a parameter of the caller rather than a constant?

Reply:
done